extensions/blocks/opentable/view.scss

```diff
@@ -1,4 +1,9 @@
 .wp-block-jetpack-opentable {
+	position: relative; /* For sake of amp-iframe with layout=fill */
```

Comment:
On my end, in reader mode, the iFrame still takes over the whole page. Is that a mode we could support as well?

Reply:
@jeherve Can you confirm that your SASS-built `view.css` for this block doesn't have this line?

I'm having trouble doing builds myself as you know, but even though I'm able to get `gulp sass:build` now to work apparently, I'm not seeing `_inc/blocks/opentable/view.css` getting updated. I've had to manually modify the CSS file to get my changes added. When I do, in Reader mode I see:

AMP Legacy chosen as Reader theme | Twenty Nineteen chosen as Reader theme
--------------------------------------|-------------------------------------------
![image](https://user-images.githubusercontent.com/134745/94201829-63f54780-fe71-11ea-86d1-5852b425ab35.png) | ![image](https://user-images.githubusercontent.com/134745/94201900-812a1600-fe71-11ea-950d-a2088a11a11a.png)

I just noticed a bug in my SASS code which I've fixed in 1652d9177d82d19b3622d4ff8a0ebcf5ecb0d627.

Reply:
It's interesting. The build is successful, but in old versions of the AMP plugin (v. 1.4.2 in my test), the CSS doesn't get added to legacy AMP views.

Reply:
Yes, this makes sense. We fixed printing block styles in legacy Reader more in v1.5. See https://github.com/ampproject/amp-wp/pull/4299.

Reply:
Can you think of a way to work around the problem on pre-1.5 sites? Right now the iFrame ends up taking over the whole page, which isn't great. I'd rather we not show the block at all in those cases. What do you think?

Reply:
Humm. There's not really a good solution for pre-1.5 sites in legacy AMP Reader mode. The reason is because before 1.5, the entire document was not sent through the post-processor, which landed in https://github.com/ampproject/amp-wp/pull/3781. This means that the stylesheets added by the Jetpack modules won't be sent through the CSS sanitizer and won't get combined into `style[amp-custom]`, resulting in invalid AMP pages.